packages/host/app/components/operator-mode/card-inheritance-panel.gts

```diff
@@ -28,8 +30,21 @@ interface Args {
 }
 
 export default class CardInheritancePanel extends Component<Args> {
-  @tracked cardInstance: CardDef | undefined;
```

Comment:
from what i can tell this was never used, weird how there was never a lint that caught this...

Reply:
I think `@tracked` breaks unused variable detection sadly